components/booking/picker_form.blade.php

```diff
@@ -6,56 +6,77 @@
-            >
-                @foreach ($__SELF__->getGuestSizeOptions() as $key => $value)
-                    <option
-                        value="{{ $key }}"
-                        {!! set_select('guest', $key) !!}
-                    >{{ $value }}</option>
-                @endforeach
-            </select>
-        </div>
-        <div class="form-group">
-            <label for="time">
-                @lang('igniter.reservation::default.label_time')
-            </label>
-            <select
-                name="time"
-                id="time"
-                class="form-control"
-            >
-                @foreach ($timeOptions as $key => $value)
-                    <option
-                        value="{{ $value->rawTime }}"
-                        {!! set_select('time', $value->rawTime) !!}
-                        {!! $value->fullyBooked ? 'disabled="disabled"' : '' !!}
-                    >{{ $value->time }}</option>
-                @endforeach
-            </select>
+<div class="form-row align-items-center progress-indicator-container">
+    @if (!$useCalendarView)
+        <div class="col-md-9 pr-md-4">
+            <div
+                data-control="datepicker"
+                data-date="{{ $__SELF__->getSelectedDate()->format('Y-m-d') }}"
+                data-start-date="{{ isset($nextOpen) ? $nextOpen->format('Y-m-d') : null }}"
+                data-days-of-week-disabled='@json($disabledDaysOfWeek ?? [])'
+                data-dates-disabled='@json($disabledDates ?? [])'
+                data-format="yyyy-mm-dd"
+            ></div>
+            <input type="hidden" name="date" value="{{ $__SELF__->getSelectedDate()->format('Y-m-d') }}"/>
         </div>
-        @if (count($timeOptions))
+        <div class="col-md-3" id="ti-datepicker-options">
+            <div class="form-group pt-4">
+                <label for="locationSelect">@lang('igniter.reservation::default.label_location')</label>
+                @partial('@input_location')
+            </div>
             <div class="form-group">
-                <button
-                    type="submit"
-                    class="btn btn-primary btn-block"
-                >@lang('igniter.reservation::default.button_find_table')</button>
+                <label for="noOfGuests">@lang('igniter.reservation::default.label_guest_num')</label>
+                @partial('@input_guest')
             </div>
-        @endif
-    </div>
+            <div class="form-group">
+                <label for="time">@lang('igniter.reservation::default.label_time')</label>
+                @partial('@input_time')
+            </div>
+            @if (count($timeOptions))
+                <div class="form-group">
+                    <button
+                        type="submit"
+                        class="btn btn-primary btn-block"
+                    >@lang('igniter.reservation::default.button_find_table')</button>
+                </div>
+            @endif
```

Comment:
Can we use something like `$this->manager->getSchedule()->isOpenAt($dateTime)` here instead?

Reply:
Again havent looked at the changes thoroughly, so this may no longer apply, but the idea was to only show the button if there were timeslots still available on that day. It may be that the restaurant is open but has no slots available?

Reply:
Ok, that makes sense.